frontend/src/i18n/locales/de.yaml

```diff
@@ -194,6 +193,49 @@ manage:
           indirekten Unterseiten sowie alle Seiteninhalte all dieser Seiten
           mitgelöscht. Die Löschung kann nicht wieder rückgängig gemacht werden!
 
+    content:
+      heading: Inhalte der Seite „{{realm}}“ bearbeiten
+      heading-root: Inhalte der Startseite bearbeiten
+
+      add: Hier einen neuen Block einfügen
+      add-text: Hier einen neuen Text-Block einfügen
+      add-series: Hier einen neuen Serien-Block einfügen
+      add-video: Hier einen neuen Video-Block einfügen
```

Comment:
Both are valid. I'm not sure what the prevalent style has been so far, but since you wrote most messages until now, and you think there shouldn't be a hyphen, it's probably what you suggested, right? :sweat_smile:

Reply:
"Fremdauthentifizierungsfehler" exists in this file :D 

I dunno. I think I would not use a hyphen here. But no blocker.

Reply:
Done in 8e3601e.